cluster/images/hyperkube/setup-files.sh

```diff
@@ -23,28 +23,36 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# Additional address of the API server to be added to the
-# list of Subject Alternative Names of the server TLS certificate
-# Should contain internal IP, i.e. IP:10.0.0.1 for 10.0.0.0/24 cluster IP range
-EXTRA_SANS=$1
+# Only create certs if they don't exist
+if [[ ! -f /data/ca.crt ]]; then
```

Comment:
How about checking each file individually before each command below? That way you'd re-generate only files that are missing. WDYT?